org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -37,8 +37,8 @@
             <repository location="http://download.eclipse.org/releases/2019-06/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-           <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.7.2.v20190528-0935"/>
-           <repository location="http://download.eclipse.org/lsp4j/updates/releases/0.7.2"/>
+            <repository location="https://github.com/akaroml/lsp4j-latest/raw/master/p2/"/>
```

Comment:
can we use http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/p2-repository/ for now?

Reply:
That would be perfect. I didn't know such a place. Let me check if it's really the latest.